com.unity.perception/Runtime/GroundTruth/PerceptionCamera.cs

```diff
@@ -104,9 +108,25 @@ void Awake()
             SetupInstanceSegmentation();
 
             RenderPipelineManager.beginCameraRendering += OnBeginCameraRendering;
+            RenderPipelineManager.endCameraRendering += CheckForRendererFeature;
             DatasetCapture.SimulationEnding += OnSimulationEnding;
         }
 
+
+        void CheckForRendererFeature(ScriptableRenderContext arg1, Camera arg2)
```

Comment:
arg1, arg2 are pretty generic names when they're used in a specific context in the actual function.  I think they should be renamed to even something like scriptableRC, and camera respectively.

Reply:
Fixed!